tests/Integration/Behaviour/Features/Scenario/Product/Combination/get_combination_ids.feature

```diff
@@ -0,0 +1,339 @@
+    Given product "product1" combination ids search criteria is set to defaults
+    When I search product "product1" combination ids by following search criteria:
+      | criteria   | value |
+      | is default | true  |
+    Then I should see following filtered combination ids of product "product1":
+      | id reference   | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SWhite | Size - S, Color - White |           | [Size:S,Color:White] | 0               | 0        | true       |
+    When I search product "product1" combination ids by following search criteria:
+      | criteria   | value |
+      | is default | false |
+    Then I should see following filtered combination ids of product "product1":
+      | id reference   | combination name        | reference | attributes           | impact on price | quantity | is default |
+      | product1SBlack | Size - S, Color - Black |           | [Size:S,Color:Black] | 0               | 0        | false      |
+      | product1SBlue  | Size - S, Color - Blue  |           | [Size:S,Color:Blue]  | 0               | 0        | false      |
+      | product1MWhite | Size - M, Color - White |           | [Size:M,Color:White] | 0               | 0        | false      |
+      | product1MBlack | Size - M, Color - Black |           | [Size:M,Color:Black] | 0               | 0        | false      |
+      | product1MBlue  | Size - M, Color - Blue  |           | [Size:M,Color:Blue]  | 0               | 0        | false      |
+
+  Scenario: I can sort combination ids by reference, quantity, impact on price
+    Given product "product1" combinations list search criteria is set to defaults
+    And I update combination "product1SWhite" with following values:
+      | impact on price | -1  |
+    And I update combination "product1SWhite" stock with following details:
+      | delta quantity  | 10  |
+    And I update combination "product1SWhite" with following values:
+      | reference       | AAA |
+    And I update combination "product1SBlue" with following values:
+      | impact on price | 1   |
+    And I update combination "product1SBlue" stock with following details:
+      | delta quantity  | 100 |
+    And I update combination "product1SBlue" with following values:
+      | reference       | BBB |
+    And I update combination "product1SBlack" with following values:
+      | impact on price | 10   |
+    And I update combination "product1SBlack" stock with following details:
+      | delta quantity  | 50 |
+    And I update combination "product1SBlack" with following values:
+      | reference       | CCC |
+    When I search product "product1" combinations list by following search criteria:
+      | criteria  | value           |
+      | order by  | impact on price |
+      | order way | asc             |
+    Then I should see following combinations in paginated list of product "product1":
```

Comment:
Is it necessary to check the list of full data as well in this scenario? Or is it to ensure that both queries return the same thing?

Reply:
its just for convenience, so we can see its sorted correctly (if I leave ids only, we won't see the price, therefore visually it won't be as clear.), but idk, I could get rid of them if you think its not worth it

Reply:
No it was just to be sure, so it makes sense to use both steps (one that checks the full data and the other only the IDs) however you need to clean the table in the steps dedicated to IDs since they are not used